File: Fundamentals of C programming/step09/9.1/9.1.04.c
/*

Определить тип символа.

Входные данные:
Один символ. Либо буква латинского алфавита, либо цифра.

Выходные данные:
digit -- если это цифра, en -- если это буква латинского алфавита. В иных случаях вывести error.

Sample Input 1:

g

Sample Output 1:

en

Sample Input 2:

5

Sample Output 2:

digit

*/

#include <stdio.h>

int main() {
  char c;
    c = getchar();
    int n = c;
    if(65 <= n && n <= 90)
        printf("en");
    else if(97<= n && n <= 122)
        printf("en");
    else if(48 <= n && n <= 57)
        printf("digit");
    else printf("error");
  return 0;
}

File: Python developer/step09/9.1.04.py
'''

Определите количество дней между двумя годами. Вам на вход поступает два числа – первый год и второй. Нужно определить количество дней между ними.

В качестве ответа выведите количество дней между годами.

Sample Input 1:

2024
2020

Sample Output 1:

1461

Sample Input 2:

2020
2024

Sample Output 2:

1461

Sample Input 3:

2020
2021

Sample Output 3:

366

Sample Input 4:

2000
1965

Sample Output 4:

12783

Sample Input 5:

1950
2023

Sample Output 5:

26663

'''

from datetime import date

d1 = date(year = int(input()), month=1, day=1)
d2 = date(year = int(input()), month=1, day=1)

delta = abs(d2 - d1)
print(delta.days)
